.github/README.md

```diff
@@ -12,99 +12,194 @@ Welcome to **Backdrop**, a command-line utility designed to manage your desktop
+   ```bash
+   nano ~/.bashrc
+   ```
+   Remove the following line (or equivalent):
+   ```bash
+   export PATH=$HOME/.backdrop/bin:$PATH
+   ```
+
+</details>
+  
+### Uninstall on Windows
+
+<details>
+<summary>Steps to Remove Installed Binary</summary>
+
+1. Open a Command Prompt and verify the location of the binary:
+   ```powershell
+   where backdrop
+   ```
+2. Remove the folder path from your PATH environment variable:
+    - Navigate to Start Menu and search for "Environment Variables".
+    - Open Edit the System Environment Variables.
+    - In the Advanced tab, click Environment Variables.
+    - Under the System Variables, select Path and click Edit.
+    - Remove the folder containing backdrop.exe from the list.
+</details>
+
+---
+
 ## &#x2705; What's Supported
 
-#### Operating Systems
-- Ubuntu/GNOME based Distros
-- CentOS/MATE (Doesn't support Slideshows)
-- Coming Soon:
-    - MacOS
-    - Microsoft (Make a PR because I won't do it)
-- Will not be Supported:
-    - WSL
+### Operating Systems
+
+- **Supported**:
+  - Ubuntu/GNOME-based Distros
+  - CentOS/MATE (Slideshows are not supported)
+  - Windows (Slideshows are not supported)
+- **Coming Soon**:
+  - macOS
+- **Not Supported**:
+  - WSL (Windows Subsystem for Linux)
+
+---
 
 ## :computer: Usage
 
 Backdrop provides several options for managing your wallpapers:
 
-- `-p, --path <PATH>`: 
-    - Set a custom path to find wallpaper images. If not provided, a default path will be used.
-- `-h, --help`: 
-    - Displays help information on how to use the command, listing all available options.
-- `-s, --slideshow`: 
-    - Will configure and set a custom slideshow of images you select with fzf. To select multiple images hit "Tab" on the images you desire to select, then hit "Enter" to confirm.
-- `-u, --url`: 
-    - You will be prompted to provide an image url to be set as wallpaper. The image will be downloaded and previewed. If confirmed, the image will be downloaded to the directory were all images are found (check "IMAGES" section). If image is NOT accepted by user, the image gets deleted and previous wallpaper is set.
-- `-v, --version`: 
-    - Print version information.
+- `-p, --path <PATH>`:
+  - Set a custom path for wallpaper images. Defaults to `$HOME/.backdrop/images` if not provided.
+- `-h, --help`:
+  - Displays help information on available commands.
+- `-s, --slideshow`:
+  - Configure and set a slideshow using images selected with `fzf`.
+- `-u, --url`:
+  - Download and set an image from a URL. Unaccepted images are deleted.
+- `-v, --version`:
+  - Print version information.
 
-For example, to set a custom path for your wallpapers, you can use the `-p` or `--path` flag:
+### Example: Set a Custom Wallpaper Directory
 
 ```bash
 backdrop --path /path/to/your/wallpapers
 ```
 
+
+---
+
 ## :busts_in_silhouette: Authors
 
-Backdrop is currently maintained by JanMichaelSE.
+Backdrop is maintained by **JanMichaelSE**.
```

Comment:
This should say `JanMIchaelSE & Theweak1` since your a main contributor to the project.

Reply:
Line corrected